docs/contribute/plans/bug-class-regression-testing.md

```
@@ -0,0 +1,521 @@
+---
+doc_type: contributor
+level: advanced
+lifecycle: active
+---
+
+# Bug-class regression testing — closing the escape pattern behind the fix-history audit
+
+**Origin:** An audit of the merged-fix history through `327df7b` (post-#883),
+covering the issue-recorded bugs, the six real-world oneAPI defects in
+#833–#838, and the escape sequences the bug-fix test contract's own
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)